Add CI for Windows (travis)

tacuaaa opened the pull request:
Closes #1667.

mforets commented on .appveyor.yml:
```diff
@@ -0,0 +1,32 @@
+  - julia_version: 1.3
+
+platform:
+  #- x86 # 32-bit
```
i have suggested to comment 32-bit as i deem unnecessary to test in that platform..

mforets commented:
@tacuaaa thanks for your contribution. I'll check what changes need to be setup in [AppVeyor](https://www.appveyor.com/). We may want to add the AppVeyor badge as well in the README.md.

schillic commented on .appveyor.yml:
```diff
@@ -0,0 +1,32 @@
+environment:
+  matrix:
+  - julia_version: 1
```
```suggestion
  - julia_version: 1.0
```
The way I understand the comment at the bottom [here](https://github.com/JuliaCI/Appveyor.jl#usage) is that `1` means "the latest version with a `1` in front" which is not what we want here.

tacuaaa commented:
I deleted .appveyor.yml, added the windows modification to .travis.yml, and also left a comment in case that in the future someone wants to test in osx.

tacuaaa commented:
All the Windows tests in travis failed, apparently they all had the same problem:
- [Julia 1](https://travis-ci.org/JuliaReach/LazySets.jl/jobs/589667349#L632)
- [Julia 1.1](https://travis-ci.org/JuliaReach/LazySets.jl/jobs/589667350#L637)
- [Julia 1.2](https://travis-ci.org/JuliaReach/LazySets.jl/jobs/589667351#L681)
- [Julia 1.3](https://travis-ci.org/JuliaReach/LazySets.jl/jobs/589667352#L3015)

I'm not quite sure, but i think that it could be fixed doing something like this: https://travis-ci.org/ararslan/Jackknife.jl/builds/568049240/config

schillic commented:
The problem is [a few lines above](https://travis-ci.org/JuliaReach/LazySets.jl/jobs/589667349#L591).

> ERROR: LoadError: UndefVarError: ffmpeg not defined

Probably some of [these commands](https://github.com/JuliaIO/FFMPEG.jl/blob/master/deps/build.jl) install it. If you look at their issue `#14`, they might also have a fix in their `master` branch. This is probably easiest to test with a Windows machine.
Note that [OpenGL also did not work](https://travis-ci.org/JuliaReach/LazySets.jl/jobs/589667349#L343). So we might as well just skip plot tests on Windows.

mforets commented:
i can't reproduce as LazySets tests work in a fresh Julia v1.2 installation in my windows machine (*) the problem. moreover, it has been reported by other people in the FFMPEG library, so we have two options:

- wait for [this issue](https://github.com/JuliaIO/FFMPEG.jl/issues/14) to be resolved
- put back appveyor CI on windows and try it (it may make a difference, idk)

(*) side note: since i had earlier data from Julia v1.1.1, i had to manually delete the entire folder `.julia/registries`, and the appropriate `.julia/environments` subfolder if it exists; then, doing `] add LazySets` and `] test LazySets` are the only commands needed).

mforets commented:
let's leave this branch for travis CI tests on windows, and create a new one to try appveyor, which should basically cherry-pick @tacuaaa's commit 7cc6600

schillic commented:
> i can't reproduce as LazySets tests work well in a fresh Julia v1.2 installation in my windows machine

You probably have installed the ffmpeg library on your computer. The Travis computers probably have a minimal setup, so you need to install everything manually.

> create a new one to try appveyor, which should basically cherry-pick @tacuaaa's commit 7cc6600

There was the problem that it did not get activated (just to say).

mforets commented:
Superseded by https://github.com/JuliaReach/LazySets.jl/pull/1739. Related to [this issue](https://travis-ci.community/t/any-chance-of-providing-an-environment-with-desktop-experience/1024) in travis ci.
